packages/editor/src/components/blocks/datatable/DataTable.tsx

```diff
@@ -43,18 +45,22 @@ export const DataTableComponent: ComponentConfig<DataTableProps> = {
     components: { subsection: 'Columns', label: 'Object-Bound Columns', type: 'selectColums' },
     paginator: { subsection: 'Paginator', label: 'Enable Paginator', type: 'checkbox' },
     maxRows: { subsection: 'Paginator', label: 'Rows per Page', type: 'number', hide: data => !data.paginator },
+    ...visibleComponentField,
     ...baseComponentFields
   },
   quickActions: ['DELETE', 'DUPLICATE', 'CREATECOLUMN']
 };
 
-const UiBlock = ({ id, components, value, paginator, maxRows }: UiComponentProps<DataTableProps>) => (
+const UiBlock = ({ id, components, value, paginator, maxRows, visible }: UiComponentProps<DataTableProps>) => (
   <>
     <div className='block-table'>
-      <div className='block-table__label'>
-        <span style={{ color: 'var(--N600)' }}>{value.length > 0 ? value : 'value is empty'}</span>
-        {paginator && maxRows !== '' && <span style={{ color: 'var(--N600)' }}>Rows per Page: {maxRows}</span>}
-      </div>
+      <UiBlockHeader
+        visible={visible}
+        label={value.length > 0 ? value : 'value is empty'}
+        labelAsIndicator={true}
+        additionalInfo={`Rows per Page: ${maxRows}`}
+      />
```

Comment:
ah because of this label you didn't solve it via the uiblockheader in the first place.
Other question, shouldn't this "label" (value is empty) be in the table part. maybe even a warning?